baseplate/experiments/__init__.py

```diff
@@ -171,66 +177,18 @@ def variant(self, name, user=None, bucketing_event_override=None,
 
         if do_log:
             assert bucketing_id
-            self._log_bucketing_event(experiment, variant, user,
-                                      extra_event_fields)
+            self._event_logger.log(
+                experiment=experiment,
+                variant=variant,
+                user_id=inputs.get('user_id'),
+                logged_in=inputs.get('logged_in'),
+                cookie_created_timestamp=inputs.get('cookie_created_timestamp'),
+                app_name=inputs.get('app_name'),
```

Comment:
i like the idea of this just being something the logger knows since it'd be created once at startup. it's a bit weird that we could vary the app_name across experiment calls.

Reply:
My concern is around services that will serve multiple apps. If relevance decides to run an experiment that changes how listings are generated, that will apply to ios, android, web, redesign. The results of those experiments may differ significantly from app to the other, and analysis should be able to bear that out.

If we created this once at startup, we'd need to create a different Experiment for each app (because we'd need a different logger for each), or else use the service as the "app_name".

I'm willing to accept that the logger is meant to know it for now (and thus remove it from the call here), but I'm trying to think ahead to how this could be handled down the line.  Either way, it's an easy addition later on if it's needed.

Reply:
Ah, I misunderstood the granularity of app_name. Sounds good.